backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -134,4 +134,16 @@ private Predicate<ConfirmationCodeEntity> byCode(String code) {
 		return confirmationCode -> confirmationCode.getCode().equals(code);
 	}
 
+    public ConfirmationCodeStatus verifyConfirmationCode(ConfirmationCode confirmationCode, User user){
```

Comment:
I'm not going to resolve this comment since the new code forces the caller to send in a `User` object when (in my opinion) a `String userId` would be preferred.

Reply:
Implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.